code/jl/american_option.jl

```diff
@@ -55,7 +55,7 @@ fontsize=16
 
 
 function plot_contours(; savefig=false, 
-                         figname="./figures/american_option_1.pdf")
+                         figname="../figures/american_option_1.pdf")
```

Comment:
@jstac we will want to keep this relative file path the same as it is currently configured in `dp-book1` (private) so the code generates figures in the current figure directly. 

So this is a good change 👍